docs/src/pages/guides/FHIRValidationGuide.md

```diff
@@ -195,11 +195,18 @@ For more information, please see: [https://confluence.hl7.org/display/FHIR/NPM+P
 The IBM FHIR server has built-in support for the following:
 
 - Profiles defined in the base FHIR Specification (v4.0.1: R4 - Mixed Normative and STU) [http://hl7.org/fhir/profilelist.html](http://hl7.org/fhir/profilelist.html)
+
+### Optional profile support
+
+The IBM FHIR server has additional implementation guides available in the fhir-validation-distribution.zip artifact, which may be downloaded from [https://github.com/IBM/FHIR/releases](releases). To load an additional implemenation guide, copy the implementation guide's jar (e.g. `fhir-ig-us-core-VERSION.jar`) from the fhir-validation-distribution.zip artifact into the IBM FHIR server's userlib directory to make it available to the server during startup.
```

Comment:
```suggestion
The IBM FHIR server has additional implementation guides available in the fhir-validation-distribution.zip artifact, which may be downloaded from [https://github.com/IBM/FHIR/releases](releases). To load an additional implemenation guide, copy the implementation guide's jar (e.g. `fhir-ig-us-core-VERSION.jar`) from the fhir-validation-distribution.zip artifact into the IBM FHIR Server's userlib directory to make it available to the server during startup.
```

Reply:
It is written as "IBM FHIR server" in other places in this guide, so I was trying to stay consistent.

Reply:
yeah we need to be consistent with IBM FHIR Server, when you see it please update